### PR TITLE
[KernelGen][Nvidia] Add batch_norm_backward_elemt operator with Triton Kernel

### PR DESCRIPTION
**Pull request by LoserCheems**
## Description

Add Triton kernel implementation for `aten::batch_norm_backward_elemt`.

This operator computes the per-element gradient for batch normalization backward pass. Given pre-computed channel-level reduction statistics (`sum_dy`, `sum_dy_xmu`), it produces `grad_input`:

```
grad_input = weight * invstd * (grad_out - (sum_dy + (input - mean) * invstd^2 * sum_dy_xmu) / count)
```

## Implementation

- **Algorithm**: 2D grid `(C, spatial_blocks)` — each program handles one channel, iterating over the batch dimension. Per-channel statistics (mean, invstd, weight, sum_dy, sum_dy_xmu) are loaded once per program instead of gathered per element.
- **Autotuning**: `libtuner` with configs varying BLOCK_SIZE (512–4096) and num_warps (4–16)
- **CUDA graph compatible**: count loaded on-device via `tl.load` (no `.item()` host sync)

## Performance

Tested on NVIDIA H20, cudagraph mode. All shapes faster than PyTorch:

### float16

| Shape | Torch (ms) | Gems (ms) | Speedup |
|-------|-----------|----------|---------|
| (4,16,64,4) | 0.0025 | 0.0013 | **1.98x** |
| (16,16,8,48) | 0.0024 | 0.0014 | **1.73x** |
| (16,16,8,88) | 0.0027 | 0.0015 | **1.82x** |
| (16,16,128) | 0.0022 | 0.0013 | **1.73x** |
| (20,6,65536) | 0.0630 | 0.0116 | **5.41x** |

### float32

| Shape | Torch (ms) | Gems (ms) | Speedup |
|-------|-----------|----------|---------|
| (4,16,64,4) | 0.0025 | 0.0014 | **1.80x** |
| (16,16,8,48) | 0.0024 | 0.0015 | **1.62x** |
| (16,16,8,88) | 0.0027 | 0.0017 | **1.62x** |
| (16,16,128) | 0.0022 | 0.0014 | **1.57x** |
| (20,6,65536) | 0.0697 | 0.0248 | **2.82x** |

### bfloat16

| Shape | Torch (ms) | Gems (ms) | Speedup |
|-------|-----------|----------|---------|
| (4,16,64,4) | 0.0025 | 0.0013 | **1.90x** |
| (16,16,8,48) | 0.0024 | 0.0014 | **1.70x** |
| (16,16,8,88) | 0.0027 | 0.0015 | **1.75x** |
| (16,16,128) | 0.0022 | 0.0013 | **1.68x** |
| (20,6,65536) | 0.0625 | 0.0121 | **5.17x** |

## Files Changed

- `src/flag_gems/ops/batch_norm_backward_elemt.py` — kernel implementation
- `src/flag_gems/ops/__init__.py` — register import
- `src/flag_gems/__init__.py` — add to `_FULL_CONFIG`
- `tests/test_batch_norm_backward_elemt.py` — accuracy tests
- `benchmark/test_batch_norm_backward_elemt.py` — performance benchmark
- `conf/operators.yaml` — operator registry entry

### CODE REVIEW

**Review by bin913 — DISMISSED** [review]

**Review by bin913 — DISMISSED** [review]

File: src/flag_gems/ops/__init__.py [hunk 1]
```diff
@@ -1393,6 +1396,7 @@
     "argmax",
     "argmin",
     "argsort",
+    "argsort_stable",
```
Comment (bin913) [inline src/flag_gems/ops/__init__.py]: Why is this added?

File: src/flag_gems/ops/__init__.py [hunk 2]
```diff
@@ -1181,6 +1182,8 @@
 from flag_gems.ops.zeros import zero_, zeros
 from flag_gems.ops.zeros_like import zeros_like
 
+argsort_stable = argsort
```
Comment (bin913) [inline src/flag_gems/ops/__init__.py]: Why is this added?

File: conf/operators.yaml [hunk 3]
```diff
@@ -1675,6 +1675,24 @@ ops:
...
+    '
+  for:
+  - batch_norm_backward_elemt
+  labels:
+  - aten
+  - KernelGen
+  kind:
+  - NeuralNetwork
+  stages:
+  - stable: '3.0'
```
Comment (bin913) [inline conf/operators.yaml]: The `5.4` branch of the FlagGems repo has been cut. Therefore, set `5.5.0` in the yaml file for the new operator.

**Review by bin913 — APPROVED** [review]
